pkg/controller/ingressdns/controller.go

```diff
@@ -273,19 +273,19 @@ func (c *Controller) getIngressStatusInCluster(cluster, key string) (*corev1.Loa
 
 	clusterIngressObj, ingressFound, err := c.ingressFederatedInformer.GetTargetStore().GetByKey(cluster, key)
 	if err != nil {
-		runtime.HandleError(fmt.Errorf("Failed to get %s ingress from %s: %v", key, cluster, err))
+		runtime.HandleError(errors.Wrapf(err, "Failed to get %s ingress from %s", key, cluster))
 		return lbStatus, err
 	}
 	if ingressFound {
 		//TODO(shashi): Find better alternative to convert Unstructured to a given type
 		clusterIngress, ok := clusterIngressObj.(*unstructured.Unstructured)
 		if !ok {
-			runtime.HandleError(fmt.Errorf("Failed to cast the object to unstructured object: %v", clusterIngressObj))
+			runtime.HandleError(errors.Errorf("Failed to cast the object to unstructured object: %v", clusterIngressObj))
 			return lbStatus, err
 		}
 		content, err := clusterIngress.MarshalJSON()
 		if err != nil {
-			runtime.HandleError(fmt.Errorf("Failed to marshall the unstructured object: %v", clusterIngress))
+			runtime.HandleError(errors.Errorf("Failed to marshall the unstructured object: %v", clusterIngress))
```

Comment:
@shashidharatd Why is err not included in the error that is handled?

Reply:
@marun @shashidharatd should i include this err?

Reply:
Please do.

Reply:
err is missed. please include